eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/core/protocol/http/processor/ReplyMessageProcessor.java

```diff
@@ -78,7 +78,7 @@ public void processRequest(ChannelHandlerContext ctx, AsyncContext<HttpCommand>
         String localAddress = IPUtils.getLocalAddress();
```

Comment:
Add this line of code.
final String channelRemoteAddr = RemotingHelper.parseChannelRemoteAddr(ctx.channel());